src/kafka/base.rs

```diff
@@ -154,7 +276,7 @@ pub trait AlertProducer {
             .set("acks", "1")
             .set("max.in.flight.requests.per.connection", "5")
             .set("retries", "3")
-            .set("debug", "broker,topic,msg")
+            // .set("debug", "broker,topic,msg") // Seems to ignore EnvFilter directives
```

Comment:
Want to delete this line instead of having commented code?

Reply:
I'd like to keep that around somehow because I had to do some digging to figure out how to get log messages from kafka. How about this instead?

```suggestion
            // To get logs from kafka, add .set("debug", "broker,topic,msg").
            // Setting an EnvFilter directive, e.g., RUST_LOG=rdkafka=debug, doesn't work.
```